timesketch/lib/analyzers/contrib/bigquery_matcher_test.py

```diff
@@ -0,0 +1,96 @@
+"""Tests for BigQueryMatcher Plugin."""
+import sys
+import copy
+import mock
+
+from timesketch.lib.testlib import MockDataStore
+
+from timesketch.lib.emojis import EMOJI_MAP
+from timesketch.lib.analyzers.contrib import bigquery_matcher
+from timesketch.lib.testlib import BaseTest
+
+
+class TestBigQueryMatcherPlugin(BaseTest):
+    """Tests the functionality of the analyzer."""
+
+    _TEST_EMOJI = 'SKULL'
+    _TEST_TAG = 'test-tag'
+    
+    # Skip the tests if bigquery is not imported.
+    __test__ = True if 'google.cloud.bigquery' in sys.modules else False
```

Comment:
Note that this seemed to be the most convenient way to skip nosetests. I also tried https://pypi.org/project/nose-exclude/ and modifying run_tests.py but it did not seem to work with the coverage options.